stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/Decorator.kt

```diff
@@ -15,14 +15,16 @@ internal interface Decorator {
 
 internal abstract class BaseDecorator : Decorator {
     override fun <T : MessageListItem> decorate(viewHolder: BaseMessageItemViewHolder<T>, data: T) {
-        when (viewHolder) {
-            is MessageDeletedViewHolder -> decorateDeletedMessage(viewHolder, data as MessageListItem.MessageItem)
-            is MessagePlainTextViewHolder -> decoratePlainTextMessage(viewHolder, data as MessageListItem.MessageItem)
-            is OnlyMediaAttachmentsViewHolder -> decorateOnlyMediaAttachmentsMessage(viewHolder, data as MessageListItem.MessageItem)
-            is PlainTextWithMediaAttachmentsViewHolder -> decoratePlainTextWithMediaAttachmentsMessage(viewHolder, data as MessageListItem.MessageItem)
-            is OnlyFileAttachmentsViewHolder -> decorateOnlyFileAttachmentsMessage(viewHolder, data as MessageListItem.MessageItem)
-            else -> Unit
-        }.exhaustive
+        (data as? MessageListItem.MessageItem)?.let {
```

Comment:
How about `if (data !is MessageListItem.MessageItem) { return }` instead?

Reply:
:+1: